### DISCUSSION TITLE
Code editor tabs/spaces

### DISCUSSION THREAD

**Discussion by yojimbo87** [standardnotes/forum#2501] [Suggestions]
Hi,

is there a way (or support) for changing tabs to spaces in code editor?

**Comment by atmoio** [comment]
No there's no way to do this I'm afraid.

**Comment by ryanpcmcquen** [comment]
Would love to see this option. This was one of the reasons I switched to [CryptPad](https://cryptpad.fr/).

**Comment by jmason888** [comment]
Was hoping the vim editor could do it with ":se sw=4"... but not luck.
